index.html

```diff
@@ -4596,78 +4596,80 @@ <h2>Key and Signature Expiration</h2>
+generated using the old secret cryptographic material can now instead be
+generated using the new  cryptographic material and can be verified using the
+new <a>verification method</a>.
       </p>
 
       <p>
-        Section <a href="#method-operations"></a> specifies the
-        <a>DID</a> operations to be supported by a
-        <a>DID method</a> specification, including
-        <a href="#method-operations">update</a>
-        which is expected to be used to perform a verification method rotation.
+Rotation is a useful mechanism for protecting against verification method
+compromise, since frequent rotation of a verification method by the controller
+reduces the value of a single compromised verification method to an attacker.
+Performing revocation immediately after rotation is useful for verification
+methods that a controller designates for short-lived verifications, such as
+those involved in encrypting messages and authentication.
       </p>
 
       <p>
-        A controller performs a rotation when they add a new verification
-        method that is meant to replace an existing verification method after
-        some time.
+The following considerations might be of use when contemplating the use of
+<a>verification method</a> rotation:
       </p>
 
-      <p>Not all DID Methods support verification method rotation.</p>
-
-      <p>
-        Rotation is a key management process that enables the private cryptographic
-        material associated with an existing verification method to be deactivated
-        or destroyed once a new verification method has been added to the DID
-        Document. Going forward, any new proofs that a controller would have generated
-        using the old cryptographic material can now instead be generated using the new
-        material and can be verified using the new verification method.
-      </p>
-
-      <p>
-        Rotation is a useful mechanism for protecting against verification
-        method compromise, since frequent rotation of a verification method by
-        the controller reduces the value of a single compromised verification method
-        to an attacker. Performing revocation immediately after rotation is
-        useful for verification methods that a controller designates for
-        short-lived verifications, such as those involved in encrypting messages
-        and authentication.
-      </p>
-
-      <p class="note">
-        Higher security environments tend to employ more frequent verification
-        method rotation.
-      </p>
-
-      <p class="note">
-        Frequent rotation of a verification method might be frustrating for
-        parties that are forced to continuously renew or refresh associated
-        credentials.
-      </p>
+      <ul>
+        <li>
+<a>Verification method</a> rotation is a proactive security measure.
+        </li>
+        <li>
+It is considered a best practice to perform <a>verification method</a> rotation
```

Comment:
Can we insert the hedge word "generally" here: "It is generally considered..."

See https://www.strongsalt.com/encryption-key-rotation-is-useless%E2%80%8A-%E2%80%8Aheres-why/ for an explanation of why not everyone agrees that this is a good idea. (In short: the more rotation, the more chances for human error and key loss.)